Clinical-Genomics/cgweb
# Add existing samples balsamic external

This PR fixes the problem to submit orders with families that you have added existing samples to.

How to test:
0. install on clinical-api on clinical-db stage
0.1 install on cgweb on clinical-db stage
1. go to the OP
1. login
1. new order
1. add family 
1. search for sample 
1. add existing sample
1. save
1. submit

Expected outcome:
Should succeed w/o errors for order types
1. external 
1. mip
1. balsamic
1. mip + balsamic
